packages/client/src/raffles/types/patchRaffleParticipation.types.ts

```diff
@@ -0,0 +1,20 @@
+import type { Config } from '../..';
+import type { Raffle, RaffleParticipation } from './raffles.types';
+
+type Operation = {
+  value: unknown;
+  path: string;
+  op: string;
+  from: string;
+};
+
+export type PatchRaffleParticipationData = {
+  operation?: Operation[] | null;
```

Comment:
Can you check if this type is correct? I think `PatchRaffleParticipationData` should be an Operation[] (and should not be undefined) according to the spec. This definition is of an object with an `operation` property in it.

Reply:
pls double check